src/enzo/CheckOverlap.C

```diff
@@ -0,0 +1,89 @@
+/***********************************************************************
+/
+/  DETERMINE IF TWO BOXES OVERLAP
+/
+/  written by: Stephen Skory
+/  date:       September, 2012
+/  modified1:
+/
+/  PURPOSE:
```

Comment:
Needs a bit of text regarding the purpose of this file.

Reply:
Done